artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/ActiveMQServerImpl.java

```diff
@@ -2508,7 +2508,7 @@ private void deployQueuesFromListCoreQueueConfiguration(List<CoreQueueConfigurat
             // if the address::queue doesn't exist then create it
             try {
                createQueue(SimpleString.toSimpleString(config.getAddress()), config.getRoutingType(),
-                           queueName, SimpleString.toSimpleString(config.getFilterString()),null,
+                           queueName, SimpleString.toSimpleString(config.getFilterString()), SimpleString.toSimpleString(config.getUser()),
```

Comment:
I don't think this is being persisted though

Reply:
@clebertsuconic it does, it was existing functionality internally, just not set-able from broker.xml I have added a test just to prove that it is persisted and survives restart

Reply:
Cool.  I have bad memory. :)


+1 on merging this.  I’m away from computer.  But feel free to merge it yourself if you feel like. 


Will merge when I get back to a computer.

Reply:
Its good, we now have an extra test now :) who doesn't like one of those :)

Ok i will merge, cheers.